src/helpers/transactions.ts

```diff
@@ -1,5 +1,6 @@
-import moment, { Moment } from 'moment';
 import _ from 'lodash';
+// eslint-disable-next-line import/named
+import moment, { Moment } from 'moment';
```

Comment:
See my other comment
```suggestion
import moment, { Moment } from 'moment';
```